### PR TITLE
Fixes for dsp partition mount

### PR DESCRIPTION
**Pull request by lumag**

### CODE REVIEW

File: recipes-support/fastrpc/fastrpc/mount-dsp.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+modprobe socinfo || true
+
```
Comment (bryanodonoghue) [inline recipes-support/fastrpc/fastrpc/mount-dsp.sh:6]: Tested-by: Bryan O'Donoghue <bryan.odonoghue@linaro.org>

File: recipes-support/fastrpc/fastrpc/mount-dsp.sh [hunk 2]
```diff
@@ -13,7 +13,14 @@ if [ -r /sys/devices/soc0/machine ] ; then
 	esac
 fi
 
-if [ -z "$WHAT" -o ! -r "$WHAT"] ; then
+if [ -z "$WHAT" -o ! -r "$WHAT" ] ; then
```
Comment (ndechesne) [inline recipes-support/fastrpc/fastrpc/mount-dsp.sh:16]: do we really need to keep this 'corner' case? when is that really useful?

Comment (lumag) [reply recipes-support/fastrpc/fastrpc/mount-dsp.sh:16]: Which corner case? No dspso image? Back then we added it for all the boards which do not have dspso.bin file for whatever reason. So I think it is still useful.

**Comment by lumag** [comment]
> Shoudln't this be SM8250|QRB5160|QRB5165 ?

There is no QRB5160 as far as I remember. it was purely my mistake.

File: recipes-support/fastrpc/fastrpc/mount-dsp.sh (hunk 2 above)
Comment (ndechesne) [reply recipes-support/fastrpc/fastrpc/mount-dsp.sh:16]: but now dspso.bin is part of the firmware release. so when are we supposed to fallback?

Comment (lumag) [reply recipes-support/fastrpc/fastrpc/mount-dsp.sh:16]: On rb3/db410c/etc

Comment (ndechesne) [reply recipes-support/fastrpc/fastrpc/mount-dsp.sh:16]: lol.. sorry.. i forgot about these other boards :)

**Pull request merged**